src/components/Navbar/index.tsx

```diff
@@ -1,40 +1,83 @@
 import React, { ReactElement } from "react"
 import { ReactComponent as ProfileIcon } from "../../assets/profile.svg"
 import { ReactComponent as StarIcon } from "../../assets/star.svg"
+import { gql, useQuery } from "@apollo/client"
 interface NavbarProps {
   Logo: ReactElement
 }
 export default function Navbar({ Logo }: NavbarProps) {
   const [searchValue, setSearchValue] = React.useState("")
 
+  const handleSearch = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchValue(e.target.value)
+    refetch({
+      filter: {
+        name: e.target.value,
+      },
+    })
+    if (data) {
+      console.log(data.characters.info.count, data.characters.results)
+    }
```

Comment:
Don't forget to remove your console log codes before sending your PRs to review.